.changeset/dry-dancers-pretend.md

```diff
@@ -0,0 +1,29 @@
+---
+'@chainlink/alongside-adapter': minor
+'@chainlink/bank-frick-adapter': minor
+'@chainlink/cfbenchmarks-test-adapter': minor
+'@chainlink/coinapi-test-adapter': minor
+'@chainlink/coingecko-test-adapter': minor
+'@chainlink/coinmarketcap-test-adapter': minor
+'@chainlink/coinmetrics-test-adapter': minor
+'@chainlink/coinpaprika-test-adapter': minor
+'@chainlink/cryptocompare-test-adapter': minor
+'@chainlink/dar-adapter': minor
+'@chainlink/dxfeed-test-adapter': minor
+'@chainlink/elwood-adapter': minor
+'@chainlink/finage-test-adapter': minor
+'@chainlink/galaxy-adapter': minor
+'@chainlink/gsr-adapter': minor
+'@chainlink/intrinio-test-adapter': minor
+'@chainlink/kaiko-test-adapter': minor
+'@chainlink/ncfx-adapter': minor
+'@chainlink/nft-blue-chip-adapter': minor
+'@chainlink/nomics-test-adapter': minor
+'@chainlink/oanda-adapter': minor
+'@chainlink/polkadot-balance-adapter': minor
+'@chainlink/tiingo-test-adapter': minor
+'@chainlink/tradermade-test-adapter': minor
+'@chainlink/twosigma-adapter': minor
+---
+
+Update v3 framework to latest version
```

Comment:
Think the changeset is missing `blocksize-capital-test` and `moonbeam-address-list`

Reply:
done